src/typebox.ts

```diff
@@ -26,6 +26,8 @@ THE SOFTWARE.
 
 ---------------------------------------------------------------------------*/
 
+import { __BaseType, __Brand } from '@coderspirit/nominal-symbols'
```

Comment:
For compatibility with Deno, this line should be replaced by, this requires post-processing 😢 .
```typescript
import { __BaseType, __Brand } from 'https://deno.land/x/nominal_symbols@1.0.2/nominal-symbols/deno/index.ts'
```